server/views/applications/show.njk

```diff
@@ -42,6 +43,19 @@
         </div>
       </div>
 
+      {% if application.assessmentDecision %}
+        {% set html %}
+        <p>Application was {{ application.assessmentDecision }} on {{ formatDate(application.assessmentDecisionDate) }}</p>
```

Comment:
does this need `govuk-body`? I'm not 100% when the class should be use but I tend to put it on every `<p>` but I am not sure why it isn't just automatically applied